src/components/Atoms/Text/Heading/Heading.tsx

```diff
@@ -47,7 +53,7 @@ const Heading = ({ type, color = 'black', text }: IHeadingProps) => {
   } else if (type === 'h4') {
     return (
       <h4
-        className={cx({
+        className={cx(className, {
           headingWhite: color === 'white',
           headingBlack: color === 'black',
         })}
```

Comment:
I still have some doubts if this is really helping us. There is a lot of code duplication in here. Do you have concerns using -just- `<h1 className={...}>...</h1>`, `<h2 className={...}>...</h2>`, .... ?

Reply:
Using className is an option depending on bootstrap class text-white. Refined to reuse class names from props instead.